docs/source/examples/notebooks/models/saving_models.ipynb

```diff
@@ -0,0 +1,274 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Saving PyBaMM models to file"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Models which are discretised (i.e. ready to solve/ previously solved, see [this notebook](https://github.com/pybamm-team/PyBaMM/blob/develop/docs/source/examples/notebooks/spatial_methods/finite-volumes.ipynb) for more information on the pybamm.Discretisation class) can be serialised and saved to a JSON file, ready to be read in again either in PyBaMM, or a different modelling library. \n",
+    "\n",
+    "In the example below, we build and solve a basic DFN model, and then save the model out to `sim_model_example.json`, which should have appear in the 'models' directory."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%pip install pybamm -q    # install PyBaMM if it is not installed\n",
+    "import pybamm\n",
+    "\n",
+    "# do the example\n",
+    "dfn_model = pybamm.lithium_ion.DFN()\n",
+    "dfn_sim = pybamm.Simulation(dfn_model)\n",
+    "dfn_sim.solve([0, 3600])\n",
```

Comment:
This is a very minor comment, but I wonder if the example (and the warning) should encourage the use of `sim.build` rather than `sim.solve`? I have a feeling many users aren't aware you can even call `sim.build`, but it seems like you might want to create a model and save it without actually running a simulation. 

I can imagine a workflow where people end up just doing a dummy solve for a short time before saving a model as they think they need to solve it first.